applications/matter_bridge/README.rst

```diff
@@ -530,6 +565,125 @@ After building the sample and programming it to your development kit, complete t
+               | Index |      Address      |                   UUID
+               ---------------------------------------------------------------------
+               | 0     | c7:44:0f:3e:bb:f0 | 0xbcd1 (Led Button Service)
+
+         #. Write down the value for the desired Bluetooth LE device index.
+            This is going to be used in the next steps (*<bluetooth_device_index>*).
+         #. Using the terminal emulator connected to the bridge, run the following :ref:`Matter CLI command <matter_bridge_cli>` to add a new bridged device:
+
+            .. parsed-literal::
+               :class: highlight
+
+               uart:~$ matter_bridge add *<bluetooth_device_index>*
+
+            The *<bluetooth_device_index>* is a Bluetooth LE device index that was scanned in the previous step.
+            For example, if you want to add a new Bluetooth LE bridged device with index ``0``, use the following command:
+
+            .. code-block:: console
+
+               uart:~$ matter_bridge add 0
+
+            The terminal output is similar to the following one:
+
+            .. parsed-literal::
+               :class: highlight
+
+               I: Added device to dynamic endpoint 3 (index=0)
+               I: Added device to dynamic endpoint 4 (index=1)
+               I: Created 0x100 device type on the endpoint 3
+               I: Created 0x103 device type on the endpoint 4
+
+         #. Write down the value for the `0x103` bridged device dynamic endpoint ID.
+            This is going to be used in the next steps (*<bridged_light_switch_endpoint_ID>*).
+
+#. Use the :doc:`CHIP Tool <matter:chip_tool_guide>` ("Writing ACL to the ``accesscontrol`` cluster" section) to add proper ACL for the light bulb device.
+   Use the following command, with *<bridge_node_ID>* and *<light_bulb_node_ID>* values from the previous step about commissioning:
+
+   .. parsed-literal::
+      :class: highlight
+
+      chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 3, "authMode": 2, "subjects": [<bridge_node_ID>], "targets": [{"cluster": 6, "endpoint": 1, "deviceType": null}, {"cluster": 8, "endpoint": 1, "deviceType": null}]}]' <light_bulb_node_ID> 0
+
+#. Write a binding table to the bridge to inform the device about all endpoints by running this command:
+
+   .. parsed-literal::
+      :class: highlight
+
+      chip-tool binding write binding '[{"fabricIndex": 1, "node": <light_bulb_node_ID>, "endpoint": 1, "cluster": 6}]' <bridge_node_ID> <bridged_light_switch_endpoint_ID>
+
+#. Depending on the chosen bridged devices configuration, complete the steps in one of the following tabs:
+
+   .. tabs::
+
+      .. group-tab:: Testing with simulated bridged light switch device
+
+         a. Using the terminal emulator connected to the bridge, run the following :ref:`Matter CLI command <matter_bridge_cli>` to add a new bridged device:
```

Comment:
I'm guessing this should be something else, since it doesn't match the instructions for the BLE side.

Reply:
Ah right, I must have made some copy+paste error. It's fixed.